homeworks/01-bubble-sort/src/helpers/generate_cols.ts

```diff
@@ -0,0 +1,19 @@
+import generateRandomValue from "./generate_random_value";
+
+import CONFIG from '../config';
+
+type Col = {
+  id: number;
+  value: number;
+};
+
+export default function generateCols(): Array<Col> {
+    const newColsLength: number = generateRandomValue(
+      CONFIG.MIN_COLS,
+      CONFIG.MAX_COLS
+    );
+    
+  return new Array(newColsLength)
```

Comment:
Just a suggestion to simplify a bit:
```ts
Array.from({ length }).map((c, id) => ({ id, value: generateRandomValue() }));
```
or even:
```ts
Array.from({ length }, (c, id) => ({ id, value: generateRandomValue() }));
```

And why `c`? It's not readable. `item`, `value` or just `_` if you don't use it.